test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+import traceback
+import unittest
+
+from configparser import ConfigParser
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+
+
+def assert_exception_correct(got: Exception, expected: Exception):
+    """
+    Compare raised exception with expected exception.
+
+    Args:
+        got (Exception): Exception received
+        expected (Exception): Exception expected
+    """
+    err = "".join(traceback.TracebackException.from_exception(got).format())
+    assert got.args == expected.args, err
+    assert type(got) == type(expected)
+
+
+class ImplTest(unittest.TestCase):
+    def setUp(self):
+        config_file = os.environ["KB_DEPLOYMENT_CONFIG"]
+        token = os.environ.get("KB_AUTH_TOKEN", None)
+
+        self.cfg = {}
+        config = ConfigParser()
+        config.read(config_file)
+        for nameval in config.items("GenomeFileUtil"):
+            self.cfg[nameval[0]] = nameval[1]
+        self.cfg["KB_AUTH_TOKEN"] = token
+
+    def test_invalid_catalog_param_type(self):
```

Comment:
I'd either 
* parameterize this in a loop and add a view more test cases
* split it into multiple tests

Reply:
Updated!

Reply:
:+1: